solr/core/src/test/org/apache/solr/rest/schema/analysis/TestManagedSynonymFilterFactory.java

```diff
@@ -64,7 +65,7 @@ private void after() throws Exception {
       jetty = null;
     }
     if (null != tmpSolrHome) {
-      FileUtils.deleteDirectory(tmpSolrHome);
+      PathUtils.deleteDirectory(tmpSolrHome.toPath());
```

Comment:
This one could also be declared as a path, I think

Reply:
Addressed in 863cf2c87c67092840b68c51908fc7a5c0aa29b0